src/Prefix.jl

```diff
@@ -615,6 +615,15 @@ function Pkg_add(args...; kwargs...)
     end
 end
 
+function Pkg_add(ctx::Pkg.Types.Context, rest...; kwargs...)
+    @static if VERSION >= v"1.13-"
+        # No need to pass the context anymore.
+        _Pkg_add(rest...; kwargs...)
```

Comment:
This needs fixing in Pkg, but as a workaround this might work?
```suggestion
        _Pkg_add(rest...; ctx.julia_version, kwargs...)
```

Reply:
I can add it if still need, but it doesn't look like it helps on `master` 😞